site/content/en/docs/Installation/Creating Cluster/eks.md

```diff
@@ -41,6 +41,14 @@ For Agones to work correctly, we need to allow UDP traffic to pass through to ou
 * Select **Inbound Rules**
 * **Edit Rules** to add a new **Custom UDP Rule** with a 7000-8000 port range and an appropriate **Source** CIDR range (`0.0.0.0/0` allows all traffic)
 
+{{% feature publishVersion="1.43.0" %}}
+## Use with custom CNI plugins
+
+To ensure compatibility with custom CNI plugins (such as Calico or [Cilium](https://docs.cilium.io/en/stable/installation/k8s-install-helm/)), configure extensions application to use `hostNetwork` for proper functionality.
```

Comment:
```suggestion
To ensure compatibility with custom CNI plugins (such as Calico or [Cilium](https://docs.cilium.io/en/stable/installation/k8s-install-helm/)), configure extensions application to use `hostNetwork` for proper functionality via the `agones.extensions.hostNetwork` Agones [Helm]({{ ref "something-I-can't-do-off-the-top-of-my-head"}}) variable. Also make sure to set the `agones.extensions.http.port` and the `agones.extensions.webhooks.port` values as well.
```

Would be good to explain _how_ you do this explicitly, as it may not be obvious - made a suggestion above. Might even want to show the full `helm` command as well.